crowbar_framework/app/models/api/node.rb

```diff
@@ -57,6 +60,7 @@ def os_upgrade
 
     # Execute post upgrade actions: prepare drbd and start pacemaker
     def post_upgrade
+      save_node_action("post-upgrade cleanup")
```

Comment:
This also includes DRBD sync which takes quite some time. We should rename the action if we know the node is DRBD controller...